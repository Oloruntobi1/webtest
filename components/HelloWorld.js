const HelloWorld = ({ style }) => <h1 style={style}>HelloWorld</h1>;

export { HelloWorld };
