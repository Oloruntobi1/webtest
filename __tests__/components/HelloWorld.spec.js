import React from 'react';
import renderer from 'react-test-renderer';

import { HelloWorld } from '../../components/HelloWorld';

const style = {
  color: 'red',
  fontSize: '14px',
};

it('renders correctly', () => {
  const tree = renderer
    .create(<HelloWorld style={style} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
