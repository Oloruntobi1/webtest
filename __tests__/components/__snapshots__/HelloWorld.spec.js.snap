// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1
  style={
    Object {
      "color": "red",
      "fontSize": "14px",
    }
  }
>
  HelloWorld
</h1>
`;
